shared/desktop/renderer/preload-main.shared.desktop.tsx

```diff
@@ -0,0 +1,8 @@
+// import Electron from 'electron'
```

Comment:
👾

Reply:
i think i won't bother merging this until the next pr is done (which includes this anyways)